core/domain/feedback_services.py

```diff
@@ -720,6 +720,10 @@ def _get_all_recipient_ids(exploration_id, thread_id, author_id):
     batch_recipient_ids = owner_ids - sender_id
     other_recipient_ids = participant_ids - batch_recipient_ids - sender_id
 
+    # Remove anonymous users from recipient list.
```

Comment:
Hm. Can we go a step back and remove None from get_all_thread_participants() instead?

Basically, None should not show up in any list.